indico/modules/events/roles/blueprint.py

```diff
@@ -26,3 +26,5 @@
 
 _bp.add_url_rule('/<int:role_id>/members/import', 'add_members_import_csv', RHEventRoleMembersImportCSV,
                  methods=('GET', 'POST'))
+_bp.add_url_rule('/<int:role_id>/members.csv', 'members_export_csv', RHEventRoleMembersExportCSV,
+                 methods=('GET',))
```

Comment:
that's the default, no need to specify it

Reply:
thanks, I had a hunch it would be the case ;)